etc/docker/node-init.sh

```diff
@@ -35,15 +58,15 @@ nsenter -t 1 -m -u -n -i apt-get update -y && nsenter -t 1 -m -u -n -i apt-get i
     bridge-utils \
     ethtool \
     curl \
-    python3.7 \
+    python$pyversion \
     lcov \
-    python3.7-dev \
+    python$pyversion-dev \
     python3-apt \
     libcmocka-dev \
     python3-pip && \
 nsenter -t 1 -m -u -n -i  update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.6 1 && \
-nsenter -t 1 -m -u -n -i  update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.7 2 && \
-nsenter -t 1 -m -u -n -i  update-alternatives --set python3 /usr/bin/python3.7 && \
+nsenter -t 1 -m -u -n -i  update-alternatives --install /usr/bin/python3 python3 /usr/bin/python$pyversion 2 && \
+nsenter -t 1 -m -u -n -i  update-alternatives --set python3 /usr/bin/python$pyversion && \
 nsenter -t 1 -m -u -n -i  ln -snf /usr/lib/python3/dist-packages/apt_pkg.cpython-36m-x86_64-linux-gnu.so /usr/lib/python3/dist-packages/apt_pkg.so && \
```

Comment:
Script got failed here
![image](https://user-images.githubusercontent.com/59909227/122364933-dc5be580-cf77-11eb-8a1a-4b19af1808e9.png)

Reply:
Yes ,this PR has 2 open issues which i had discussed with Amit. We will be raising separate PR to mizar repo from other forked vrsion. 
This is draft PR which needs to be tested thoroughly.
Thanks for review though